### DISCUSSION TITLE
Feature Request: Toggle for gestures in ebook reader

### DISCUSSION THREAD

**Discussion by RWoof** [gotson/komga#1654] [Ideas]
When reading ebooks, swiping left or right will jump a page back or forward. This can get annoying when you are trying to scroll to read and unintentionally it jumps to another spot in the book. It would be nice to be able to disable the gesture, at least when you're not at the top or bottom of the page.

**Comment by gotson** [comment]
Agreed, i was not aware of this behaviour. It's a default behaviour in https://github.com/d-i-t-a/R2D2BC the reader Komga uses. Ideally this should be configurable, and/or disabled when using scroll mode.
